src/pages/Folder/Folder.jsx

```diff
@@ -0,0 +1,72 @@
+import Header from "components/Header/Header";
+import Footer from "components/Footer/Footer";
+import styles from "./Folder.module.css";
+import AvatarImg from "assets/Avatar.png";
+import SearchIcon from "assets/Search.png";
+import { useEffect, useState } from "react";
+import CardList from "components/CardList/CardList";
+import { getFolderInfo } from "api/api";
+import { useAsync } from "hooks/useAsync";
```

Comment:
P3:
component 폴더에서 각 항목별로 개별 import 를 하니 가독성이 좀 떨어지는 것 같아요.
아래와 같은 식으로 export 하고 import 할 수 있습니다.

```suggestion
import { Header, Footer, CardList } from "components"; // component들은 한 폴더에서 여래개 export 될 수 있으니 named export 활용
import styles from "./Folder.module.css";
import AvatarImg from "assets/Avatar.png";
import SearchIcon from "assets/Search.png";
import { useEffect, useState } from "react";
import { getFolderInfo } from "api";
import { useAsync } from "hooks";
```